### PR TITLE
fix the delete method path

### PR DESCRIPTION
**Pull request by kavehmz**
It seems during refactoring path for delete and get which were separate where unified wrongly which this PR should fix it.

### CODE REVIEW

**Comment by vicente-ramos** [comment]
@kavehmz Looks good to me.

File: bin/cleaner.rb [hunk 1]
```diff
@@ -5,9 +5,16 @@
 require 'ike_artifactory'
 
 actually_delete = false
-if ARGV[0] == '--actually-delete'
-  ARGV.shift
-  actually_delete = true
+delete_empty_path = false
+while ARGV[0] =~ /^--/ do
+  arg = ARGV.shift
```
Comment (nickmarden) [inline bin/cleaner.rb:10]: So I think this says "if there are any arguments before `--actually-delete` and `--delete-empy-path`, throw those arguments away and ignore them"

**Review by nickmarden — CHANGES_REQUESTED** [review]
If we are going to write `bin/cleaner.rb` the way it is suggested, we should not throw away arguments that come before the `--actually-delete` and `--delete-empty-path` switches.

File: bin/cleaner.rb (hunk 1 above)
Comment (nickmarden) [reply bin/cleaner.rb:10]: Oh wait, I misread this. Please ignore my comment.

**Review by nickmarden — APPROVED** [review]

File: bin/cleaner.rb [hunk 2]
```diff
@@ -5,9 +5,16 @@
...
+  if arg == '--actually-delete'
+    actually_delete = true
+  end
+  
+  if arg == '--delete-empty-path'
```
Comment (nickmarden) [inline bin/cleaner.rb:15]: Need to update the usage statement.

**Review by nickmarden — APPROVED** [review]

**Pull request merged**